README.md

```diff
@@ -1,5 +1,5 @@
-pkg - a binary package manager for FreeBSD
-============================================
+pkg - a binary package manager for FreeBSD and OpenBSD
+========================================================
```

Comment:
Please, don't add this change unless this is the offical position of OpenBSD developers.